### DISCUSSION TITLE
Low 1-Year Mortality - Using dod Field

### DISCUSSION THREAD

**Discussion by georgeasrian** [MIT-LCP/mimic-code#1414] [MIMIC-IV]
(Similar to [this discussion post](https://github.com/MIT-LCP/mimic-code/discussions/1263))

I'm noticing in MIMIC-IV there is a very low mortality per the "dod" field in the patients module. In this project I'm analyzing the proportion of patients with hip fracture diagnoses that have passed way within 1 year of hospital admission. The rate, manually calculated using the "dod" field, is extremely low (~5%) compared to what it typically rate is which is at least ~25%. Is there any reason this "dod" field may not be reliable for my purposes?

**Comment by alistairewj** [comment]
Which version did you use? MIMIC-IV v2.0 added out of hospital mortality and was released June 2022. It may be you were using the older version (v1.0) which did not have out of hospital mortality.
